RELEASE_PROCESS.md

```diff
@@ -145,11 +176,16 @@ do the following:
     rm -rf jujugui-deps/*
     make package
 
+Also modify the `metadata.yaml` file by removing the line starting with
+"series: ".
+
 At this point `jujugui-deps` should have no wheels, only gzipped tarballs.
 After testing and QA, upload the fat charm to the charmstore with:
 
-    charm2 upload --publish . cs:~yellow/precise/juju-gui
-    charm2 upload --publish . cs:~juju-gui-charmers/precise/juju-gui
+    charm push . cs:~yellow/precise/juju-gui
+    charm push . cs:~juju-gui-charmers/precise/juju-gui
+
+And then publish them with `charm publish` specifying the revisions.
```

Comment:
I guess the stable one, used if no channel is specified when calling "charm publish".